xbdd-web/src/modules/report/ReportContainer.jsx

```diff
@@ -1,41 +1,33 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
 import { Grid, Card } from "@material-ui/core";
-import TagListContainer from "./tag-list/TagListContainer";
-import Report from "../../models/Report";
-import { getBuildReport } from "../../lib/rest/Rest";
+import FeatureListContainer from "./FeatureListContainer/FeatureListContainer";
 
 class ReportContainer extends Component {
   constructor(props) {
     super(props);
-    this.state = { report: null };
+    this.state = {
+      selectedFeature: null,
+      report: null,
+    };
   }
 
   componentDidMount() {
-    getBuildReport(this.props.product, this.props.version, this.props.build).then(data => {
-      this.setState({
-        report: new Report(data),
-      });
-    });
+    // Fetch data for report or stat
```

Comment:
Is this going to be utilised or is it not getting used for anything? If you are going to put stuff in here later it's fine but let's delete it for now if you're not sure

Reply:
Yes, the next thing is to handle when a feature is clicked here and render the report next to the feature list.